public/locales/auset-te.json

```diff
@@ -0,0 +1,426 @@
+      "body": "ఒక సమూహంగా, రూట్‌బోటిక్స్ చట్టంలో వివరించిన విధంగా వాటి సెటప్ సూచనలను అనుసరించి, మీరు ఏ బాట్‌లతో ఆడాలనుకుంటున్నారో నిర్ణయించుకోండి."
+    },
+    "chooseLandmarks": {
+      "title": "ల్యాండ్‌మార్క్‌లను సెటప్ చేయండి",
+      "body": "సమూహంగా, మీరు ఎన్ని ల్యాండ్‌మార్క్‌లతో ఆడాలనుకుంటున్నారో మరియు ఎంపిక పూల్‌లో ఏవి చేర్చాలో నిర్ణయించుకోండి.",
+      "body_mapLandmark": "సమూహంగా, మీరు <i>(ఇప్పటికే చేర్చబడిన మ్యాప్ ల్యాండ్‌మార్క్‌తో పాటు)</i>తో ఎన్ని ల్యాండ్‌మార్క్‌లను ప్లే చేయాలనుకుంటున్నారో మరియు ఎంపిక పూల్‌లో ఏవి చేర్చాలో నిర్ణయించుకోండి."
+    },
+    "chooseHirelings": {
+      "title": "కూలీని సెటప్ చేయండి",
+      "body": "సమూహంగా, మీరు అద్దెకు తీసుకునే వారితో ఆడాలనుకుంటున్నారా మరియు ఎంపిక పూల్‌లో ఏవి చేర్చాలో నిర్ణయించుకోండి."
+    },
+    "postHirelingSetup": {
+      "subtitle": "ప్లేస్ హైర్లింగ్ మార్కర్స్",
+      "body": "స్కోర్‌లోని \"4,\" \"8,\" మరియు \"12\" ఖాళీలపై \"4,\" \"8,\" మరియు \"12\" - మార్క్ చేసిన మూడు నియామక గుర్తులను ఉంచండి మ్యాప్‌లో ట్రాక్ చేయండి."
+    },
+    "drawCards": {
+      "title": "5 కార్డులు తీసుకోండి",
+      "body": "ప్రతి క్రీడాకారుడు షేర్డ్ డెక్ నుండి ఐదు కార్డులను డ్రా చేస్తాడు. <i>(మీరు తర్వాత ఉంచడానికి మూడు కార్డ్‌లను ఎంచుకుంటారు.)</i>"
+    },
+    "chooseFactions": {
+      "title": "వర్గాలను సెటప్ చేయండి",
+      "body": "సమూహంగా, మీరు ఫ్యాక్షన్ డ్రాఫ్ట్‌లో ఏ వర్గాలను చేర్చాలనుకుంటున్నారో ఎంచుకోండి."
+    },
+    "selectFaction": {
+      "subtitle": "ఫ్యాక్షన్ ఎంచుకోండి",
+      "body": "ప్లేయర్ {{count}}, దిగువన ఉన్న ఫ్యాక్షన్ పూల్ నుండి మీరు ఏ ఫ్యాక్షన్‌ని ఆడాలనుకుంటున్నారో ఎంచుకోండి. \"తదుపరి దశ\"ని నొక్కడం ద్వారా మీ ఎంపికను నిర్ధారించే ముందు మీరు వారి గణాంకాలను చూడటానికి జాబితా నుండి వర్గాలను ఉచితంగా ఎంచుకోవచ్చు."
+    },
+    "placeScoreMarkers": {
+      "title": "స్కోర్ మార్కర్లను పెట్టండి",
+      "title_vagabondSetUp": "స్కోర్ మరియు రిలేషన్ షిప్ మార్కర్స్ ను పెట్టండి",
+      "body": "ప్రతి క్రీడాకారుడు స్కోర్ ట్రాక్ యొక్క \"0\" స్థలంలో వారి ఎంపిక చేసిన వర్గం యొక్క స్కోర్ మార్కర్‌ను ఉంచుతారు.",
+      "body_vagabondSetUp": "ప్రతి క్రీడాకారుడు స్కోర్ ట్రాక్ యొక్క \"0\" స్థలంలో వారి ఎంపిక చేసిన వర్గం యొక్క స్కోర్ మార్కర్‌ను ఉంచుతారు. ప్రతి వాగాబాండ్ ఆటగాడు రిలేషన్ షిప్‌ల ఉదాసీన ప్రదేశంలో ఆడుతున్న ప్రతి నాన్-వాగాబాండ్ వర్గానికి రిలేషన్ షిప్ మార్కర్‌ను ఉంచుతాడు."
+    },
+    "chooseHand": {
+      "title": "ప్రారంభ చేతులు ఎంచుకోండి",
+      "body": "ప్రతి క్రీడాకారుడు తమ చేతిలో మూడు కార్డ్‌లను ఉంచడానికి ఎంచుకుంటాడు మరియు మిగిలిన రెండు కార్డ్‌లను షేర్డ్ డెక్‌లో ఉంచాడు. పూర్తయిన తర్వాత, షేర్డ్ డెక్‌ని షఫుల్ చేయండి."
+    },
+    "setupEnd": {
+      "title": "ఆట ప్రారంభించండి",
+      "body": "సెటప్ ఇప్పుడు పూర్తయింది. ప్లేయర్ {{count}}తో ఆట ప్రారంభమవుతుంది. మీరు సెటప్ ప్రాసెస్‌ను పునఃప్రారంభించాలనుకుంటే, దిగువ బటన్‌ను ఉపయోగించండి.<p><i>\"రూట్ ఆటోమేటెడ్ సెటప్\" ఎవెన్ కామెరాన్ ద్వారా డెవలప్ చేయబడింది</i><br/><i>బోర్డ్ గేమ్ ఆధారంగా \"రూట్ \" Leder Games ద్వారా ప్రచురించబడింది</i><br/><i>CC BY-NC-SA 4.0</i></p> కింద లైసెన్స్ పొందిన అప్లికేషన్"
```

Comment:
`<i>CC BY-NC-SA 4.0</i></p> కింద లైసెన్స్ పొందిన అప్లికేషన్`
The final line spills outside the paragraph. `</i></p>` should be _after_ the final piece of text

Also, feel free to add a credit for yourself here for work on the translation 😄. The translated line with the above fix and your credit (where it says "YOUR CREDIT HERE") should look like this:
`సెటప్ ఇప్పుడు పూర్తయింది. ప్లేయర్ {{count}}తో ఆట ప్రారంభమవుతుంది. మీరు సెటప్ ప్రాసెస్‌ను పునఃప్రారంభించాలనుకుంటే, దిగువ బటన్‌ను ఉపయోగించండి.<p><i>\"రూట్ ఆటోమేటెడ్ సెటప్\" ఎవెన్ కామెరాన్ ద్వారా డెవలప్ చేయబడింది</i><br/><i>YOUR CREDIT HERE</i><br/><i>బోర్డ్ గేమ్ ఆధారంగా \"రూట్ \" Leder Games ద్వారా ప్రచురించబడింది</i><br/><i>CC BY-NC-SA 4.0 కింద లైసెన్స్ పొందిన అప్లికేషన్</i></p>`

See the [French translation](https://github.com/EwenDC/root-automated-setup/blob/4d1e4ded732b7b5898b641c9d3fc6e4493687d50/public/locales/auset-fr.json#L125) for an example